### PR TITLE
Divide logits by 300 in KNN image classifier for numerical stability.

### PR DESCRIPTION
**Pull request by nsthorat**
This PR also:

* Fixes gitignore to not ignore index.ts.
* Updates library version.
* Adds index.ts for each of the models, which was not checked in in the last PR.

---
This change is [Reviewable](https://reviewable.io/reviews/pair-code/deeplearnjs/334)

### CODE REVIEW

**Comment by jameswex** [comment]
Reviewed 7 of 7 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/pair-code/deeplearnjs/334)*

**Comment by jameswex** [comment]
:lgtm_strong:

---

Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/pair-code/deeplearnjs/334#-:-KyaPSxlBsSpg4QWN3il:b-jbsf8z)*
